app/controllers/api/works_controller.rb

```diff
@@ -1,11 +1,13 @@
 class Api::WorksController < ApplicationController
   respond_to :json
+
   def show
     @work = Work.find(params[:id])
     render json: @work
   end
 
   def create
+    authorize! :create, Work
```

Comment:
Does not use `load_and_authorize_resource` method due to complications due to the post params' nested attributes. In addition, using `load_and_authorize_resource` for now may make development more difficult when other developers implement methods and run into permissions complications. I think we should keep it this way for now, and then revisit in the future.

Reply:
This is ok! Some people may prefer the more explicit access control definitions and it also indirectly gives u more fine grained control. The nested attributes thing does look strange to me though

Reply:
Yeah :// its because originally we were using paperclip, which only allows one image to be associated with each entry in the db, so we had to set up a has-many relation to deal with that. I think @calvind777 is fixing this in his newest PR